nio_cli/nio_add_blocks/main.py

```diff
@@ -118,7 +118,30 @@ def __init__(self, args):
         self.args = args
 
     def perform(self):
+        ''' Clone down the project_template repository
+
+        If the installed version of nio matches a release in the
+        project_template repo, then use that tag.
+        '''
         BlockAdder.add(self.args.project_name, True, self.args.https)
+        version = self._nio_version()
+        print('n.io version: {}'.format(version))
```

Comment:
We ok printing here? Normally I assume these are left over accidentally but with the CLI we may want this?